packages/feedback/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,20 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule, NoPreloading } from '@angular/router';
+import { FeedbackHomeComponent } from './modules/feedback/home/feedback-home.component';
+import { AppComponent } from './app.component';
+
+const routes: Routes = [{
+    path: '',
+    redirectTo: 'feedback',
```

Comment:
Don't redirect to `/feedback`. As the App is already going to be hosted inside a `feedback` directory on Apache server.
The route inside the app should just be the root.

Having said that, @sayak-sarkar can you confirm/validate if I'm right here?

Reply:
I agree @deshmukhmayur, the route within the app should just be the root.